packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -27,7 +26,7 @@ function sleep(period) {
   });
 }
 
-describe('ReactTestUtils.act()', () => {
+describe('React.act()', () => {
```

Comment:
```suggestion
describe('ReactTestUtils.act()', () => {
```

name was fine. I thought it spelled `ReactDOMTestUtils `

Reply:
React.act() is probably clearer even though it no longer matches the filename. Just don't want to rename to make blaming easier.